packages/Tag/src/tag.scss

```diff
@@ -72,11 +72,79 @@
   --ids-tag-border-has-error: #{tokens.$coral-600};
 }
 
+[data-brand="workleap"] {
+  /* Default */
+  --ids-tag-border-radius: var(--hop-shape-rounded-sm);
+  --ids-tag-border-radius-rounded: var(--hop-shape-pill);
```

Comment:
both radius-rounded and radius-rounded-medium would have the same radius?

Reply:
yes for workleap they do from what I can tell, but the old Igloo had different values.